packages/calypso-e2e/package.json

```diff
@@ -21,10 +21,18 @@
 	},
 	"devDependencies": {
 		"@types/config": "^0.0.38",
-		"@types/node": "^15.0.2"
+		"@types/jest": "^25.2.3",
+		"@types/node": "^15.0.2",
+		"jest": "^26.6.3",
+		"ts-jest": "^26.5.6"
```

Comment:
Why `ts-jest` instead of `@babel/preset-typescript`?

Looks like `ts-jest` also gives us type checking, but we already have a CI job for that (as long as your TS package is declared in `./packages/tsconfig.json`)

Reply:
I followed a tutorial I found that suggested using `ts-jest`. No deeper reason than that, so not hung up on sticking with `ts-jest`. I will try getting this to work with `@babel/preset-typescript`.